### DISCUSSION TITLE
Mouth Masking Option ($75)

### DISCUSSION THREAD

**Discussion by hacksider** [hacksider/Deep-Live-Cam#735] [Bounty]
This basically just adds opacity on the lower region of the face. Allowing you to freely move your tongue or mouth without affecting quality of the deepfake.
![image](https://github.com/user-attachments/assets/a3e201f6-0e29-40ac-82e2-c8dda11f8772)

If you need more idea or clue on how to get this done, feel free to comment below. 

Payment via paypal/alipay

**Comment by ineow1** [comment]
does it still retain the deepfake lips and skin or is it just separated from the rest of the face? and is this going to be free feature down the line?

**Reply by hacksider** [reply]
Everything is a free feature, it will be included on the main repository (why not?).

**Reply by hacksider** [reply]
It is separated from the deepfake. This is to ensure there won't be any artifacts.

**Comment by MW0973** [comment]
Is it not already implemented in one of the forks, https://github.com/iVideoGameBoss/iRoopDeepFaceCam ?

**Reply by hacksider** [reply]
Yes it is already implemented there. This is basically a giveaway for those that have enough time (the time of the implementation is Aug 27). 😄

**Comment by hacksider** [comment]
Rewarded to @KRSHH 
https://github.com/hacksider/Deep-Live-Cam/pull/740
